microsoft-edge/devtools-guide-chromium/whats-new/2024/08/devtools-128.md

```diff
@@ -14,11 +14,11 @@ ms.date: 08/22/2024
 
 
 <!-- ====================================================================== -->
-## Updated labels for selector stats
+## Updated labels for CSS selector stats
 
 <!-- Subtitle: Use the "Enable CSS selector stats" setting instead of the "Enable advanced paint instrumentation (slow)" to capture CSS selector statistics for Recalculate Style events -->
 
-In the **Performance** tool, the duplicate **Selector Stats** label has been removed from the `Recalculate Style` event details.  Selector stats can be enabled by using the dedicated setting for it.
+In the **Performance** tool, the duplicate **Selector Stats** label has been removed from the `Recalculate Style` event details.  Selector stats can be enabled by using the dedicated setting<!-- todo: what is the ui path? eg: the Foo checkbox --> for it.
```

Comment:
where/what specifically is "the dedicated setting for it"?  
@leahmsft

Reply:
the checkbox is named "Enable CSS selector stats (slow)" in the Perf panel

Reply:
result:
```
In the **Performance** tool, the duplicate **Selector Stats** label 
has been removed from the `Recalculate Style` event details.  To 
enable selector stats, use the **Enable CSS selector stats (slow)** 
checkbox in the **Performance** tool.
```